routing/pathfind.go

```diff
@@ -656,33 +653,32 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 		// of the private channel, we'll assume it was selected as a
 		// routing hint due to having enough capacity for the payment
 		// and use the payment amount as its capacity.
-		bandWidth := partialPath.amountToReceive
+		bandWidth := distance[pivot].amountToReceive
```

Comment:
Hmmm... is this line changed because we now have pointers and need an up to date version of `nodeWithDist`? In that case, are we still putting the right data structure on the heap? Or should it be more minimal with just the pubkey on the heap?

Reply:
I am wondering whether there is a pre-existing bug here?

Reply:
I took another look and indeed, `distance[pivot]` isn't changing as you say.